website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -6,115 +6,63 @@ weight: 3
+- Access to a Kubernetes v1.11.3+ cluster.
 
-```sh
-$ mkdir memcached-operator && cd memcached-operator
-$ operator-sdk init --plugins=ansible --domain=example.com
-$ operator sdk create api --group=cache --version=v1alpha1 --kind=Memcached --generate-role
-```
-
-This creates the memcached-operator project specifically for watching the
-Memcached resource with APIVersion `cache.example.com/v1alpha1` and Kind
-`Memcached`.
-
-To learn more about the project directory structure, see [project
-layout][layout-doc] doc.
-
-#### Operator scope
+## Creating an Operator
 
-Read the [operator scope][operator-scope] documentation on how to run your operator as namespace-scoped vs cluster-scoped.
+In this section we will:
+  - extend the Kubernetes API with a [Custom Resource Definition][custom-resources] that allows users to create `Memcached` resources.
+  - create a manager that updates the state of the cluster to the desired state defined by `Memcached` resources.
 
-## Customize the operator logic
+#### Scaffold a New Operator
 
-For this example the memcached-operator will execute the following
-reconciliation logic for each `Memcached` Custom Resource (CR):
-- Create a memcached Deployment if it doesn't exist
-- Ensure that the Deployment size is the same as specified by the `Memcached`
-CR
+Begin by generating a new project from a new directory.
 
-## Watch the Memcached CR
-
-By default, the memcached-operator watches `Memcached` resource events as shown
-in `watches.yaml` and executes Ansible Role `Memcached`:
-
-```yaml
----
-- version: v1alpha1
-  group: cache.example.com
-  kind: Memcached
+```sh
+$ mkdir memcached-operator 
+$ cd memcached-operator
+$ operator-sdk init --plugins=ansible --domain example.com
 ```
 
+Among the files generated by this command is a Kubebuilder `PROJECT`
+file. Subsequent `operator-sdk` commands (and help text) run from the
+project root read this file and are aware that the project type is
+Ansible.
 
-#### Options
-
-**Role**
-Specifying a `role` option in `watches.yaml` will configure the operator to use
-this specified path when launching `ansible-runner` with an Ansible Role. By
-default, the `new` command will fill in an absolute path to where your role
-should go.
-```yaml
----
-- version: v1alpha1
-  group: cache.example.com
-  kind: Memcached
-  role: memcached
+```sh
+# Since this is an Ansible-based project, this help text is Ansible specific.
+$ operator-sdk create api -h
 ```
 
-**Playbook**
-Specifying a `playbook` option in `watches.yaml` will configure the operator to
-use this specified path when launching `ansible-runner` with an Ansible
-Playbook
-```yaml
----
-- version: v1alpha1
-  group: cache.example.com
-  kind: Memcached
-  playbook: playbook.yaml
-```
+Next, we will create a `Memcached` API.
 
-See [watches reference][ansible-watches] for more information.
+```sh
+$ operator-sdk create api --generate-playbook --group cache --version v1alpha1 --kind Memcached --generate-role
```

Comment:
Do we need both flags `--generate-playbook` `--generate-role` for the example. Are we not using just the role here? 
Also, I think is better add them at the end to make clear GKV --options

Reply:
I think this suggestion landed in the wrong spot.